src/Faker/Provider/fa_IR/Person.php

```diff
@@ -134,4 +134,82 @@ class Person extends \Faker\Provider\Person
 
     protected static $titleMale = array('آقای', 'استاد', 'دکتر', 'مهندس');
     protected static $titleFemale = array('خانم', 'استاد', 'دکتر', 'مهندس');
+
+    /**
+     * this method returns a valid Iranian nationalCode
+     * @return string
+     * @example '8075859741'
+     */
+    public static function nationalCode()
```

Comment:
Please document this new method in the locale-specific section of the README. Also, please unit test it.

Reply:
You got it.

Reply:
As a good practice, please place the @return phpDoc annotation as last in the list.

Reply:
You got it.